src/virtual-scroller.js

```diff
@@ -1,9 +1,7 @@
-import {_item, _key, ItemSource} from './item-source.js';
-import {default as Layout1dGrid} from './layouts/layout-1d-grid.js';
-import {default as Layout1d} from './layouts/layout-1d.js';
-import {VirtualScroller} from './virtual-scroller.js';
-
-export {ItemSource};
```

Comment:
why not exporting here?

Reply:
It seemed odd that `ItemSource` was threaded through this file when other pieces like layouts weren't, so I figured it would be more consistent to say that all pieces you want should imported explicitly.